min-max/min-max-sum/chandan-min-max-sum.js

```diff
@@ -0,0 +1,57 @@
+class MaxMin {
+    constructor() {
+        console.log("Constructor initialized!");
+        this.arr = arr;
+    }
+    getMin() {
+        var min = this.arr[0];
+        for(var i=1; i<this.arr.length; i++) {  
+            if(min > this.arr[i]) {
+                min = this.arr[i];
+            }
+        }
+        return min;
+    }
+    getMax() {
+        var max = this.arr[0];
+        for(var i=1; i<this.arr.length; i++) {  
+            if(max < this.arr[i]) {
+                max = this.arr[i];
+            }
+        }
+        return max;
+    }
```

Comment:
add `getter` and `setter` for `arr` property.

Reply:
would you please explain a bit?